### ISSUE TITLE
Option to disable/hide temporary unblocking buttons

### ISSUE THREAD

**Issue by TheDcoder** [raicem/impulse-blocker#36]
It would be nice to have an option to disable those temporary unblocking buttons, I find them counter-productive as I would be deep in surfing and I get suddenly hit by the blocked message, resisting to click the button again doesn't work very well and it has sort of developed like a habit within me.

The method of manually unblocking is much better since you are in control and can be demotivated to unblock again since you voluntarily blocked it before.

We already have an option to hide the on/off buttons, something similar for the temporary time-based buttons would be good too :smile:

**Comment by raicem** [comment]
That sounds good to me. It should be very easy to implement since we have the underlying logic for hiding on/off buttons as you have mentioned. 

I'll implement this once I get into the code. I know there are couple of bugs I need to resolve as well. Many thanks for the idea!

**Comment by raicem** [comment]
Feature is implemented in the latest commit. It will be in the release 1.1.0.
